sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+    "    endpoint_name=llama_endpoint_name,\n",
+    "    model=foundation_model.id,\n",
+    "    instance_type=compute_sku_for_llama,\n",
+    "    instance_count=compute_instance_count_for_llama,\n",
+    "    request_settings=OnlineRequestSettings(\n",
+    "        request_timeout_ms=60000,\n",
+    "    )\n",
+    ")\n",
+    "ml_client.online_deployments.begin_create_or_update(demo_deployment).wait()\n",
+    "# deployment takes 100 traffic\n",
+    "llama_endpoint.traffic = {deployment_name: 100}\n",
+    "ml_client.online_endpoints.begin_create_or_update(llama_endpoint)\n",
+    "\n",
+    "llama_endpoint = ml_client.online_endpoints.get(name=llama_endpoint_name)\n",
+    "print(llama_endpoint)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6. Create `score.py` for the Safety enabled LLaMa endpoint\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 6.1 Create a folder to save the score.py and conda dependencies file.\n",
+    "First create a source folder for the score.py file and conda dependencies file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "scoring_src_dir = \"./safety-llama\"\n",
+    "os.makedirs(scoring_src_dir, exist_ok=True)\n",
+    "print(f\"Scoring script directory: {scoring_src_dir}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 6.2 Create the score.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 6.3 Create the conda.yaml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile {scoring_src_dir}/conda.yaml\n",
+    "name: aacs-conda\n",
+    "channels:\n",
+    "  - defaults\n",
+    "dependencies:\n",
+    "  - python=3.9\n",
+    "  - pip:\n",
+    "    - azure-identity==1.14.0b1\n",
+    "    - azure-ai-ml==1.8.0\n",
+    "    - azureml-inference-server-http==0.8.4\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 7. Create a Managed Identity for the safety enabled LLaMa endpoint\n",
+    "As you can see in the above steps, we specified auth_mode to aml_token for the LLaMa online endpoint and Content Safey is support token based authentication by default, which means we need to create a managed identity for the safety enabled LLaMa endpoint, so that it can access the Azure AI Content Safety and the LLaMa endpoint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 7.1 Decide on the name of your user identity:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "uai_name = f\"{endpoint_name}-uai\"\n",
+    "print(f\"Will create UAI {uai_name}\")"
```

Comment:
Do we need to add an explanation for UAI? Not sure if customers can understand this concept clearly.

Reply:
explanation added.